src/Illuminate/Queue/Events/JobQueued.php

```diff
@@ -5,21 +5,21 @@
 class JobQueued
 {
     /**
-     * @var mixed
+     * @var  string|object
      */
     public $job;
 
     /**
-     * @var mixed
+     * @var  string|int|null
```

Comment:
Please don't add extra spaces in var docs, only param. If in doubt, please look at what the rest of the codebase does.